variants/m5stack_stickc_plus/m5stack_nanoc6/pins_arduino.h

```diff
@@ -0,0 +1,45 @@
+#ifndef Pins_Arduino_h
```

Comment:
Currently this file is in `variants/m5stack_stickc_plus/m5stack_nanoc6/pins_arduino.h` and should be in `variants/m5stack_nanoc6/pins_arduino.h`